src/main/resources/application.yaml

```diff
@@ -12,13 +12,13 @@ management:
         include: health, info, prometheus
 
 springdoc:
-  packagesToScan: uk.gov.hmcts.reform.demo.controllers
+  packagesToScan: uk.gov.hmcts.reform.libra-dtu.controllers
   writer-with-order-by-keys: true
 
 #If you use a database then uncomment below lines and update db properties accordingly
 spring:
   config:
-    import: "optional:configtree:/mnt/secrets/rpe/"
+    import: "optional:configtree:/mnt/secrets/hmi/"
   application:
     name: Spring Boot Template
```

Comment:
Can the spring app name be updated too here

Reply:
updated